data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1000 @@
+			choice
+				`	"How do you know my language?"`
+					goto humans
+					to display
+						not "avgi: asked about humans"
+				`	"Tell me more about the Avgi."`
+					goto avgi
+					to display
+						not "avgi: asked about avgi"
+				`	"That's all the questions I have. What happens now?"`
+					goto conclusion
+
+			label conclusion
+			`	Prasinos says, "Assurance. Of course, we will introduce your fellow humans to you. But..." he trails off.`
+			`	Asime continues for him. "Apprehension. The humans that came here before you, I must confess are still here for a reason. Perhaps they can give more details, but they have told us that when they first entered our space, they immediately attempted to leave. For whatever reason, they were unable to. I do not know the means by which you were able to travel to our space, but perhaps you will be able to return, and take these stranded humans with you."`
+			`	"Or, you may find yourself trapped here, just as they are. In any case, we will of course make you as comfortable as we can, as an honored guest, and treat you as we would any foreign dignitary."`
+			choice
+				`	"I have a jump drive that lets me travel beyond the hyperlanes, but it won't let me jump back to human space."`
+				`	"I have a device that lets me go where I please, but it isn't working at the moment."`
+
+			`	"Regret. Then the situation is most unfortunate," she says. "Commitment. We will offer you what help we can, but I did not wish to lie to you about their predicament, or provide you with false hope."`
+			`	"Reservation. Perhaps Captain <last> will be able to find a solution." says Galano. "I would not dismiss the possibility too quickly."`
+			choice
+				`	"While I'm here, can I buy your technology?"`
+					goto buystuff
+				`	"Is there anything I can do to help in the meantime?"`
+
+			`	"Consideration. As an alien, you will inevitably be a novelty to most of us, but the existence of humans is well-known among the public enough for you to walk among us without issue. I will speak to others about a more formal role, but until then I am unsure if we can ask anything of you beyond keeping the peace."`
+			label buystuff
+			`	Galano says, "Interest. I would expect human technology to be superior to our own in most aspects, but we will of course allow you to purchase items with your human credits, which we will back with our own currency with the agreement of the Assembly. In the event that it becomes possible to open trade relations with your human government, we may find it useful to have a small stockpile of your currency available."`
+			`	"Amusement. Not to mention the goodwill such a gesture could buy us," says Asime.`
+			`	Prasinos says, more seriously, "Reservation. Our material resources are quite limited at the moment, so for now I would only support providing for the requisition of small items that do not strain the needs of the Guard. No military goods, at least not now."`
```

Comment:
They comment here that they would not provde military goods, yet several of their weapons are immediately available for purchase. I feel like there should either be a clarification as to which weapons are meant by this or all the weapons should be put behind a license.